podtato-services/hats/docker/Dockerfile

```diff
@@ -31,10 +31,12 @@ FROM scratch
 
 COPY --from=builder /dist/podtatoserver /
 COPY /static  /static
-COPY /static/podtato-new.html  /podtato-new.html
+
+ARG VERSION
+ENV VERSION=${VERSION}
 
 # Export necessary port
 EXPOSE 9000
 
 # Command to run when starting the container
-CMD ["/podtatoserver", "--version", "0.1.1"]
\ No newline at end of file
+CMD ["/podtatoserver"]
```

Comment:
afaik the version args were added in the last few weeks ...

Reply:
yes I have moved it to env var instead of flags, this makes building all the images for diff versions easier